app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Service Catalog
+subtitle: Track every service across your architecture
+content-type: explanation
+---
+
+{{site.konnect_saas}}'s Service Catalog offers a comprehensive catalog of all services running in your organization. This catalog gives you insight into your organization's services and their dependencies. It includes both {{site.base_gateway}} services and external integration resources.
+
+<!-- vale off-->
+{% mermaid %}
+graph LR
+  %% Define styles for icons and nodes
+  classDef integration fill:#f7f7f7,stroke:#333,stroke-width:2px;
+  classDef resource fill:#e1f7d5,stroke:#333,stroke-width:2px;
+  classDef konnect fill:#d0e5ff,stroke:#333,stroke-width:2px;
+  classDef serviceCatalog fill:#e1f7d5,stroke:#333,stroke-width:2px;
+  classDef billingInventory fill:#f0f0f0,stroke:#333,stroke-width:2px;
+
+  %% Define the resources
+  GitHub[<img src="https://img.icons8.com/ios/50/000000/github.png" width="24" height="24"/> GitHub]:::resource
+  PagerDuty[<img src="https://play-lh.googleusercontent.com/E-zhAf4KJ6JDDXmQfQxBprn2sATGYUMkOEqLQX5HAQQtiwDZJg4c8sQd7deb6nCZCwU=w480-h960-rw" width="24" height="24"/> PagerDuty]:::resource
+  GatewayManager[<img src="https://raw.githubusercontent.com/Kong/docs.konghq.com/main/app/assets/images/icons/kong-gradient.svg" width="15" height="15"/> Gateway Manager]:::resource
```

Comment:
Kong icon way too big can we fix that somehow?

Reply:
We can, you would have to set a max size on the icon. The styling code here isn't actually doing anything (looks AI-generated?). When you adjust the source for the icons, you should also do something like:

```
style="max-height:50px; max-width:50px" class="no-image-expand"
```

You also want that `no-image-expand` class because otherwise the icons will be clickable (images expand by default into a modal).